dags/atd_knack_signals.py

```diff
@@ -0,0 +1,112 @@
+import os
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+from pendulum import datetime, duration, now
+
+from utils.onepassword import get_env_vars_task
+from utils.slack_operator import task_fail_slack_alert
+from utils.knack import get_date_filter_arg
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
```

Comment:
Add a `description`, here's the one from the airflow v1 DAG

```
"description": "Load signals (view_197) records from Knack to Postgrest to AGOL and Socata",  # noqa:E501
```

Reply:
I moved the description down to the `DAG` params? this seems like the right place 🤷

Reply:
oops yes that's much better
![](https://giphy.com/explore/knowing-youre-right)

Reply:
thanks @Charlie-Henry for flagging the change in date filter behavior. as far as i can know, the proper truncate/replace behavior (with no date filter) is going to be preferred. iirc, the reason i initially wrote these the  '1970-01-01'  way is because you can't pass `None` to a CLI arg (`--date`) 🤷 

that said, when we deploy to prod, i think we'll want to monkey patch in today's date for `prev_start_date_success` - otherwise the first time these DAGs run they will all run in replace mode, and there are some real big datasets out there.